ynh_add_fail2ban_config/ynh_add_fail2ban_config

```diff
@@ -42,9 +42,10 @@ EOF
   ynh_store_file_checksum "$finalfail2banjailconf"
   ynh_store_file_checksum "$finalfail2banfilterconf"
   
-..sudo systemctl restart fail2ban
-..if local fail2ban_error="$(journalctl -u fail2ban | tail -n50 | grep "WARNING Command.*$app.*addfailregex")"
-..then
+  systemctl restart fail2ban
+  local fail2ban_error="$(journalctl -u fail2ban | tail -n50 | grep "WARNING.*$app.*")"
```

Comment:
Why did you removed "Command." and ".*addfailregex" for the grep command ?

Reply:
I had other WARNING that were interesting to show (among with the `regrexp` typo 😄)... ideally there shouldn't be any remaining warnings.